service/service.go

```diff
@@ -1,29 +1,32 @@
 package service
 
 import (
-	"encoding/hex"
 	"fmt"
+	"os"
+	"path/filepath"
+
 	"github.com/btcsuite/btcd/btcec"
 	"github.com/medibloc/panacea-doracle/config"
 	"github.com/medibloc/panacea-doracle/crypto"
+	"github.com/medibloc/panacea-doracle/event"
 	"github.com/medibloc/panacea-doracle/panacea"
 	"github.com/medibloc/panacea-doracle/sgx"
 	"github.com/medibloc/panacea-doracle/types"
-	"os"
-	"path/filepath"
+	log "github.com/sirupsen/logrus"
 )
 
 type Service struct {
-	Conf          *config.Config
-	OracleAccount *panacea.OracleAccount
-	OraclePrivKey *btcec.PrivateKey
-	UniqueID      string
```

Comment:
I think it would be better to leave the `UniqueID` alone. This is because there is an inconvenience of having to encode in hex every time to use this.

Reply:
Yeah. I think so, but also don’t wanna have a duplicate value along with enclaveInfo. 
Maybe we can define a `enclaveInfo.UniqueIDHex()`. I think the performance of the hex encoding is not a big issue. So, we can improve the convenience by defining that function.

Reply:
Yep. That seems to be the better way.